blockonomics-woocommerce.php

```diff
@@ -60,29 +60,19 @@ function blockonomics_woocommerce_init()
     add_action('init', 'load_plugin_translations');
     add_action('woocommerce_order_details_after_order_table', 'nolo_custom_field_display_cust_order_meta', 10, 1);
     add_action('woocommerce_email_customer_details', 'nolo_bnomics_woocommerce_email_customer_details', 10, 1);
-    add_action( 'restrict_manage_posts', 'filter_orders' , 20 );
-    add_filter( 'request', 'filter_orders_by_address_or_txid' );	
+    add_action('admin_enqueue_scripts', 'blockonomics_load_admin_scripts' );
     add_filter('woocommerce_payment_gateways', 'woocommerce_add_blockonomics_gateway');
     add_filter('clean_url', 'bnomics_async_scripts', 11, 1 );
 
+
     /**
-     * Adding new filter to WooCommerce orders
+     * Add Styles to Blockonomics Admin Page
      **/
-    function filter_orders() {
```

Comment:
It already has merged *origin/master* . Github will not bring in such changes by itself. This is a mistake during conflict resolution

@cnohall  please be careful with conflict resolution. This is very critical. This will potentially create/remove major sections of code after are unwanted. You may want to read more about using git

Reply:
I've rebased the PR. I'll be more careful in conflict resolution and will study git more closely so that it doesn't happen again.